internal/bminventory/inventory.go

```diff
@@ -3177,3 +3177,142 @@ func proxySettingsForIgnition(httpProxy, httpsProxy, noProxy string) (string, er
 	}
 	return buf.String(), nil
 }
+
```

Comment:
Can we move all of this out of inventory.go?  The file is very bloated already and this PR doesn't relate to inventory.
See internal/events/ for example.

Reply:
Yes. Done.